crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -444,6 +444,8 @@ en:
       identify: 'Identify'
     form:
       allocate: 'Allocate'
+      confirm_allocate: 'WARNING! Allocating the node will wipe all the disks attached to the node including the shared disks. Proceed?'
```

Comment:
I'm not a UI expert, but I think it's generally wrong to have a whole uppercase word (even for WARNING).